chain.go

```diff
@@ -13,6 +13,11 @@ type (
 	}
 )
 
+// NewChain - Create an instance of Chain
+func NewChain(drivers []Cache) *Chain {
```

Comment:
I have been thinking that the NewChain should use variadic function instead Cache array/slice, it's more easy to use.